iModelCore/ECDb/Tests/NonPublished/SchemaUpgradeTests.cpp

```diff
@@ -10945,20 +11021,21 @@ TEST_F(SchemaUpgradeTestFixture, KoQDeleteWithDoNotFailFlag)
             <KindOfQuantity typeName="KoQ1" displayLabel="KoQ1" relativeError=".5" persistenceUnit="u:CM" />
         </ECSchema>)schema")));
 
+    auto schema = m_ecdb.Schemas().GetSchema("TestSchema");

```

Comment:
Should we be changing this test?  It's covering the DoNotFail flag so it should be a case where it should fail.  I know we want to get rid of this flag but we have not yet so we should leave the tests as is.  If this is testing something unique with your modifications then you should create a new test for that case as I expect this test will be removed when we remove that flag

Reply:
Yup, makes sense.
I've updated the code and the tests.